src/4D_api.cpp

```diff
@@ -2069,6 +2069,7 @@ PJ  *proj_create_crs_to_crs_from_pj (PJ_CONTEXT *ctx, const PJ *source_crs, cons
             }
         }
 
+        P->over = forceOver;
```

Comment:
To simplify finding future issues with `git bisect`, thoughts about moving the force over fix to a separate PR?

Reply:
> thoughts about moving the force over fix to a separate PR?

it is needed in this PR, otherwise tests in gie_self_test break (or I would have to create a PR with this fix only, and rebase that one on top of it. Not sure if that changes much about bisection processes? at least this is a separate commit). To be honest, I don't fully understand why my changes have some interaction with the forceOver stuff. Well probably that the fact that now we look up transformations we cannot instantiate (to be able to warn about their lack) has revealed an underlying bug in the forceOver stuff that didn't pop up before.